src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        Ball secondBall = Lottery.getRandomBall();
+        Ball thirdBall = Lottery.getRandomBall();
+
+        Ball[] balls = {firstBall, secondBall, thirdBall};
```

Comment:
Use a for loop for creating several objects of the same class and adding them to an array. This will make your code more concise and avoid the repetition of similar lines of code.